tests/driver_dac_dds/main.c

```diff
@@ -171,9 +178,17 @@ static void play_function(uint16_t period, uint32_t samples, sample_gen_t fun)
     static uint8_t buf[DAC_BUF_SIZE];
     mutex_t lock = MUTEX_INIT_LOCKED;
 
+    if (period > DAC_BUF_SIZE) {
+        printf("Period duration exceeds sample buffer size.\n");
+        return;
+    }
+
     /* only work with whole wave periods */
     uint16_t len_aligned = DAC_BUF_SIZE - DAC_BUF_SIZE % period;
 
+    /* One underrun indication is expected (for the first sample) */
+    int32_t underruns = -1;
```

Comment:
might as well use an `int` here, then you don't have to use those ugly `PRId32` format strings static tests wants you to use.

Reply:
Good point. An earlier version printed the samples too, but now that it doesn't, why bother.

Reply:
... and fixup'd.

Reply:
please squash!